lib/browser/guest-view-manager.js

```diff
@@ -319,29 +320,71 @@ const watchEmbedder = function (embedder) {
   })
 }
 
-ipcMain.on('ELECTRON_GUEST_VIEW_MANAGER_CREATE_GUEST', function (event, params, requestId) {
+const handleMessage = function (channel, handler) {
+  ipcMain.on(channel, (event, ...args) => {
+    if (event.sender._isWebViewTagEnabled()) {
```

Comment:
If this is firing for a lot of events (E.g. a lot of async calls) should we memoize `_isWebViewTagEnabled`?

On a related not we don't need to expose a new API for this, we can use `event.sender.getLastWebPreferences().webviewTag`

Reply:
reimplemented as suggested